Microsoft.DotNet.Interactive.Formatting/Formatter.cs

```diff
@@ -19,6 +19,12 @@ public static class Formatter
         private static int _recursionLimit;
         internal static readonly RecursionCounter RecursionCounter = new RecursionCounter();
 
+        private static string _defaultMimeType = HtmlFormatter.MimeType;
```

Comment:
I'm not sure this is the correct default.

Reply:
what should we set then?